astroquery/eso/core.py

```diff
@@ -1,59 +1,55 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+=====================
+ESO Astroquery Module
+=====================
+European Southern Observatory (ESO)
+"""
 
 import base64
 import email
+import functools
 import json
+import os
 import os.path
+import sys
 import re
 import shutil
 import subprocess
 import time
 import warnings
-import webbrowser
 import xml.etree.ElementTree as ET
-from io import BytesIO
-from typing import List, Optional, Tuple, Dict, Set
+from typing import List, Optional, Tuple, Dict, Set, Union
 
 import astropy.utils.data
 import keyring
-import requests.exceptions
+import requests
 from astropy.table import Table, Column
 from astropy.utils.decorators import deprecated_renamed_argument
 from bs4 import BeautifulSoup
+from pyvo.dal import TAPService
+from pyvo.dal.exceptions import DALQueryError, DALFormatError
 
 from astroquery import log
 from . import conf
-from ..exceptions import RemoteServiceError, NoResultsWarning, LoginError
+from ..exceptions import RemoteServiceError, LoginError, \
+    NoResultsWarning, MaxResultsWarning
 from ..query import QueryWithLogin
 from ..utils import schema
-
-__doctest_skip__ = ['EsoClass.*']
-
-
-def _check_response(content):
-    """
-    Check the response from an ESO service query for various types of error
-
-    If all is OK, return True
-    """
-    if b"NETWORKPROBLEM" in content:
-        raise RemoteServiceError("The query resulted in a network "
-                                 "problem; the service may be offline.")
-    elif b"# No data returned !" not in content:
-        return True
+from .utils import _UserParams, raise_if_coords_not_valid, _reorder_columns, \
+    _raise_if_has_deprecated_keys, _build_adql_string, \
+    DEFAULT_LEAD_COLS_PHASE3, DEFAULT_LEAD_COLS_RAW
```

Comment:
have an all to keep the tab complete clean and to signal the public API.
```suggestion
    DEFAULT_LEAD_COLS_PHASE3, DEFAULT_LEAD_COLS_RAW
    
    
__all__ = ['Eso', 'EsoClass']
```